presto-main/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -83,6 +83,7 @@
             .add("com.facebook.drift.annotations.")
             .add("com.facebook.drift.TException")
             .add("com.facebook.drift.TApplicationException")
+            .add("org.objectweb.asm.")
```

Comment:
Why do we need this library in the SPI?

Reply:
Good catch. I think this was a remnant from the drift-codec excludes hack that we had initially. 
Since we went via the airlift upgrade remote I have removed it and tested server startup and it looks good.